share/man/man9/intro.9

```diff
@@ -178,7 +179,7 @@ File system implementations register themselves with
 .Xr vfsconf 9 .
 .Pp
 The abstract and filesystem-independent representation of a file, directory, or
-other file-like entity within the kernel is the
+other file-like entity within the kernel, is the
```

Comment:
How about (in the interest of active voice):
```
The vnode(9) is the abstract and filesystem-independent representation of a file, directory, or other file-like entity within the kernel.
```

Reply:
Done with [`26786f1` (#813)](https://github.com/freebsd/freebsd-src/pull/813/commits/26786f1819fd84931b4e757900ecca5adae8d970).